zowex/src/util.rs

```diff
@@ -134,9 +140,27 @@ pub fn util_get_socket_string() -> Result<String, i32> {
 }
 
 pub fn util_get_zowe_env() -> HashMap<String, String> {
-    env::vars()
+    let mut environment: HashMap<String, String> = env::vars()
         .filter(|(k, _)| k.starts_with("ZOWE_"))
-        .collect()
+        .collect();
+
+    match env::var("FORCE_COLOR") {
+        Ok(val) => {environment.insert(String::from("FORCE_COLOR"), val);},
+        Err(_val) => {environment.extend(util_get_color_env());}
+    }
+
+    // Make sure ansi is enabled for the response
+    if !util_enable_ansi() {
+        environment.insert(String::from("FORCE_COLOR"), String::from("0"));
+    }
+
+    environment
+}
+
+pub fn util_get_color_env() -> HashMap<String, String> {
+    let mut temp_map = HashMap::new();
+    temp_map.insert(String::from("FORCE_COLOR"), String::from(util_terminal_supports_color().to_string()));
```

Comment:
`util_terminal_supports_color().to_string()` is already a `String`, can remove `String::from` here